python/ruff/__main__.py

```diff
@@ -1,6 +1,7 @@
 import os
 import sys
 import sysconfig
+from pathlib import Path
```

Comment:
`pathlib` is expensive to import during the startup, so we actively wanted to avoid using it; use os.path instead